src/python/pants/option/global_options.py

```diff
@@ -184,6 +184,16 @@ def from_options(cls, options: Options) -> ExecutionOptions:
                 remote_execution_headers = auth_plugin_result.execution_headers
                 remote_store_headers = auth_plugin_result.store_headers
 
+        # Prefix the remote servers with a scheme.
+        remote_address_scheme = "https://" if bootstrap_options.remote_ca_certs_path else "http://"
```

Comment:
Should this be `grpc` and `grpcs`? I think we were planning on users choosing from those two, and not offering `http` and `https`.

Reply:
The Tonic gRPC library requires the schemes to be either `http` or `https`. We can expose those to Pants CLI users as `grpc` and `grpcs` but these cannot be passed through as-is to Tonic since it will reject `grpc`/`grpcs` as unknown.